build.gradle

```diff
@@ -1,6 +1,28 @@
-group 'multi-int-store'
-version '0.1.0-SNAPSHOT'
+plugins {
+	id 'org.springframework.boot' version '2.1.4.RELEASE'
+	id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.connexta'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
 
 repositories {
-    mavenCentral()
-}
\ No newline at end of file
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.retry:spring-retry'
+	compileOnly 'org.projectlombok:lombok'
+	annotationProcessor 'org.projectlombok:lombok'
```

Comment:
❓ What are `org.springframework.retry:spring-retry`, `org.projectlombok:lombok`, and `org.projectlombok:lombok` used for?

Reply:
Dependencies that I thought we would end up using. Lombok is used for all the spring annotation magic. You would also need to enable the Lombok plugin in IntelliJ fyi.